ex6.4_remember.py

```diff
@@ -0,0 +1,19 @@
+from PIL import Image
+
+
+def decipher():
+    black = 1
+
+    with Image.open("code.png").convert() as img:
+        pixel = img.load()
+        w = img.size[0]
```

Comment:
Don't use abbreviations in variable names -
https://softwareengineering.stackexchange.com/questions/72593/how-to-abbreviate-variable-names
Try to avoid i and j as well if there is a better name available.